[Docs] Describe how action works and depends on each other (#5386)
DevExpress/testcafe: docs/articles/faq/README.md

2. Add the [t.debug()](../documentation/guides/basic-guides/debug.md) method before this selector to stop test execution before it reaches this point.
3. Run the test and wait until the browser stops at the breakpoint.

> After this, use the browser's development tools to check that elements satisfy the [interaction requirements](../documentation/guides/basic-guides/interact-with-the-page.md#interaction-requirements).

Reviewer: ```suggestion
Next, use the browser's development tools to check that elements satisfy the [interaction requirements](../documentation/guides/basic-guides/interact-with-the-page.md#interaction-requirements).
```
